specification/applicationinsights/resource-manager/readme.md

```diff
@@ -97,6 +97,12 @@ directive:
     where: $.definitions.WorkbookResource.properties.name
     from: workbooks_API.json
     reason: 'This property is already a part of our API, cannot remove it'
+  - suppress: DefaultErrorResponseSchema
+    from: myworkbooks_API.json
+    reason: 'this error format is already part of the previous api, cannot change it'
+  - suppress: DeleteOperationResponses
+    from: myworkbooks_API.json
+    reason: 'delete API returns empty response and is already part of the previous api, cannot change it'
```

Comment:
What's the meaning of previous apis? Consistent with `workbooks_API.json`?

Reply:
yes'ish.  the endpoints/apis for workbooks and myworkbooks are
(1) exactly the same
(2) shipped at exactly the same time

i'm not sure why the S360 about myworkbooks only started showing up.  but now i have to add that, so i've been trying to fix it up.  it appears that years ago when workbooks was added, these rules either didn't exist or didn't apply, but now they do for myworkbooks.  so it keeps showing them as failures in lintdiff.  if there's a more correct way to do this, let me know.